src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,34 @@
+/** @jsxRuntime classic /
+/** @jsx jsx */
+import { createElement } from "react"
+import * as TablerIcons from "@tabler/icons"
+
+type IconType = typeof import("@tabler/icons")
+
+export interface IconProps {
+  name: string
+
+  // set custom `width` and `height`
+  size?: number
+
+  // set `stroke` color
+  color?: string
+
+  // set the thickness stroke
+  stroke?: number
+}
+
+const Icon = ({
+  name = "IconAlertTriangle",
+  size = 20,
+  color = "blue",
```

Comment:
Maybe should be using a color from the theme as the default color

Reply:
that makes sense, gonna change it